hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileGetInfoHandler.java

```diff
@@ -81,6 +88,18 @@ public void validate(@NonNull final QueryContext context) throws PreCheckExcepti
         }
     }
 
+    public Fees computeFees(@NonNull QueryContext queryContext) {
+        final var query = queryContext.query();
+        final var fileStore = queryContext.createStore(ReadableFileStore.class);
+        final var op = query.fileGetInfoOrThrow();
+        final var fileId = op.fileIDOrThrow();
+        final File file = fileStore.getFileLeaf(fileId);
```

Comment:
It really hurts to see this. 🙂 Let's discuss this in one of our calls.

Reply:
I agree with you 100%